backend/tarsy/integrations/llm/client.py

```
@@ -8,11 +8,12 @@
 import pprint
 import traceback
 import urllib3
-from typing import Dict, List, Optional
+from typing import Any, Dict, List, Optional, Tuple
 
 # Suppress SSL warnings when SSL verification is disabled
 urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
 
+from langchain_core.callbacks import UsageMetadataCallbackHandler
```

Ruff (0.12.2)

Module level import not at top of file

(E402)